polar/src/lib.rs

```diff
@@ -313,6 +313,20 @@ pub extern "C" fn polar_question_result(query_ptr: *mut Query, call_id: u64, res
     })
 }
 
+#[no_mangle]
+pub extern "C" fn polar_error(query_ptr: *mut Query, message: *mut c_char) -> i32 {
```

Comment:
Maybe `polar_application_error` or `polar_set_error`? To make this clearly separate from the other get error function?